documentation/src/main/asciidoc/user_guide/cache_api.adoc

```diff
@@ -65,6 +65,19 @@ if (cachedPerson == null) {
 
 Please note that link:{javadocroot}/org/infinispan/Cache.html#putForExternalRead-K-V-[putForExternalRead] should never be used as a mechanism to update the cache with a new Person instance originating from application execution (i.e. from a transaction that modifies a Person's address). When updating cached values, please use the standard link:{jdkdocroot}/java/util/Map.html#put-K-V-[put] operation, otherwise the possibility of caching corrupt data is likely.
 
+==== Clearing the cache
+
+The fastest way to clear a cache is to call
+link:{javadocroot}/org/infinispan/Cache.html#clear--[clear] method.
```

Comment:
Should we recommend `clear()` here? The javadoc still says "This should never be invoked in production"